apps/image-focal-point/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div id="root"></div>
+
+    <script type="module" src="./src/index.jsx"></script>
```

Comment:
```suggestion
    <script type="module" src="./src/index.js"></script>
```
doesn't look like there's a jsx file in this app